frontend/javascripts/oxalis/controller/td_controller.js

```diff
@@ -161,42 +168,40 @@ class TDController extends React.PureComponent<Props> {
         this.setTargetAndFixPosition();
       },
       pinch: delta => this.zoomTDView(delta, true),
+      mouseMove: (delta: Point2, position: Point2) => {
+        Store.dispatch(setMousePositionAction([position.x, position.y]));
+      },
+      leftClick: (pos: Point2, plane: OrthoView, event: MouseEvent, isTouch: boolean) => {
+        if (skeletonControls != null) {
+          skeletonControls.leftClick(pos, plane, event, isTouch);
+        }
```

Comment:
In the rare case, that the current mouse position points on a isosurfaces and a node (and the correct modifiers are pressed), it can happen that both actions are executed. However, this shouldn't do any harm in my opinion.

Reply:
Yes it's probably harmless. If this becomes a frequent problem and annoys users we can think about only executing one of the two, depending on which element is nearer to the camera.